RELEASE.md

```diff
@@ -2,146 +2,259 @@
-* The job consists of 3 steps which are configured in the `Jenkinsfile`: **build**, **package** and **publish**.
-* We kept the `Jenkinsfile` to a minimum and put most of the build logic in several JavaScript files executed with node.js and prefixed with `job-`.
+* Everything is managed by some GitHub action workflows
+* There are 4 workflows
+  * Build: test and build
+  * Publish: publish packages to various targets.
+  * Release: use `release-please` to maintain a release PR
+  * Preview: handles previews when some event occurs on PR
+* We kept the workflows to a minimum and put most of the build logic in several JavaScript files executed with node.js and prefixed with `job-`.
 * The `job-*.js` scripts use some external binaries when it was easier (or when there was no strong JS based solution):
   * `tar`
   * `zip`
   * `git`
   * `npm`
   * `ssh-keyscan`
   * `fpm` (which is a ruby gem)
-* We run this Jenkins job on an agent using this Docker image: [clever-tools-builder](https://hub.docker.com/r/clevercloud/clever-tools-builder/).
-* The `Dockerfile` to build this image is here: [docker-runner/Dockerfile](./docker-runner/Dockerfile)
-* The job is triggered on each new commit and each new tag via a classic Jenkins git hook setup in the GitHub project.
-  * If it's a commit in a branch, we only run the **build** and **package** steps, and we archive artefacts afterwards.
-  * If it's a commit in a tag, we run the **build**, **package** AND **publish** steps.
-  * The version in the **publish** step uses the git tag.
 
 ### Build
 
-The first step of the job builds binaries for GNU/Linux, macOS and Windows using [pkg](https://github.com/zeit/pkg).
-This allows us to release a self-contained binary without having to worry about the inner node.js dependencies etc... and other implementation details of the project. 
+The Build workflow is triggered on every commit on a branch associated with a PR. It is also run on any commit on the `master` branch.
 
-### Package
+It does the following:
 
-The second step of the job packages various types of archives and bundles for different needs and computes SHA 256 sums:
+#### Check commit messages
+
+We use [CommitLint](https://github.com/conventional-changelog/commitlint) to enforce the commit messages format.
+All commit messages must follow the [conventional commit](https://www.conventionalcommits.org) rules.
+They will be used to generate the CHANGELOG.
+
+#### Ensure code quality
+
+We use [ESLint](https://eslint.org/) to enforce JS coding style and spot linting errors ASAP.
+
+Run this locally step with:
+```shell
+npm run lint
+```
+
+Or with auto fix:
+```shell
+npm run lint:fix
+```
+
+#### Build binaries
+
+This step does three things:
+
+* Builds binaries for GNU/Linux, macOS and Windows using [pkg](https://github.com/zeit/pkg).
```

Comment:
nitpick: not your fault as the link was like this before but the repo is https://github.com/vercel/pkg